### PR TITLE
feat(style): Add `Noto` font family

### PR DESCRIPTION
**Pull request by gretanausedaite**
Issue #625 

Add `Noto Sans` and `Noto Sans Mono` fonts

### CODE REVIEW

File: packages/itwinui-css/backstop/assets/demo.css [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
+  src: url('fonts/NotoSans-Light.ttf') format('truetype');
 }
+
 @font-face {
-  font-family: 'Open Sans';
+  font-family: 'Noto Sans';
   font-style: normal;
   font-weight: 400;
-  src: url('fonts/open-sans-v28-latin-regular.woff2') format('woff2');
+  src: url('fonts/NotoSans-Regular.ttf') format('truetype');
```
Comment (mayank99) [inline packages/itwinui-css/backstop/assets/demo.css:16]: Why `ttf`? We should be using `woff2`.

Comment (gretanausedaite) [reply packages/itwinui-css/backstop/assets/demo.css:16]: Google fonts doesn't have woff2? https://fonts.google.com/noto/specimen/Noto+Sans

**Review by veekeys — APPROVED** [review]
I have scrolled through images, mostly look fine. 
Non ideal state ones were weird, that font change also influence the width of the image.
Also input button combo image looked weird, that it influenced width and height I think.

I would agree to check `woff2` type too. t showed some results, but not sure if reliable (i.e. https://github.com/GreatWizard/notosans-fontface)

File: packages/itwinui-css/backstop/assets/demo.css (hunk 1 above)
Comment (gretanausedaite) [reply packages/itwinui-css/backstop/assets/demo.css]: I added Noto Sans woff2 files. Noto sans mono is variable font - maybe we can leave it as is?

Comment (mayank99) [reply packages/itwinui-css/backstop/assets/demo.css]: Noto Sans Mono size:
- variable ttf: 1.3MB
- variable woff2 latin full: 51KB
- variable woff2 latin weights: 28KB
- weight 400 woff2 latin: 9KB

I would be ok with any of the latter three. 1.3MB is not an acceptable size for any webfont.

Comment (gretanausedaite) [reply packages/itwinui-css/backstop/assets/demo.css]: Changed to variable woff2 latin full: 51KB

**Review by mayank99 — APPROVED** [review]

**Review by adamhock — APPROVED** [review]

**Review by gretanausedaite — COMMENTED** [review]
Changed Mono font file to more extended one because previous didn't have arrow symbols (therefore tests failing). Unfortunately, it's `woff` and 156KB

**Comment by gretanausedaite** [comment]
Could not make tests fail locally and I don't know what's wrong, adding back `woff` file.

**Pull request merged**